src/context/SearchResults.js

```diff
@@ -0,0 +1,35 @@
+import React, { createContext } from 'react';
+
+import Spinner from '../components/Spinner';
+import ErrorMessage from '../components/ErrorMessage';
+
+import useFetch from '../hooks/useFetch';
+
+export const SearchResultsContext = createContext();
+
+const SearchResults = ({ children }) => {
+  const { lat, lng } = { lat: 48.8566, lng: 2.3522 };
```

Comment:
I wonder if there is a way to set the `lat` and `lng` in only one place. Right now it's being done here, and in the `useFetch`. Maybe `useFetch` should just take in the lat and lng.

Reply:
Hi @laurenmbeatty. There's been so much refactoring going on here - we've completely missed that. Yes, we're going to have it one place only😄